### ISSUE TITLE
.NET 4.8 WPF ToolTip: Unexpected Behavior

### ISSUE THREAD

**Issue by rockerinthelocker** [dotnet/wpf#4427]
If a control (i.e. StatusBar) with attached ToolTip control has focus and the window containing the control has its ShowTaskbar property set to false, and the window is minimized, the tooltip pops up after the window is minimized.

Additionally, if a tooltip opened on keyboard focus (introduced in .NET 4.8) is closed programmatically, moving the mouse out of the window and then straight back over the control does not trigger the ToolTip until the mouse is moved over another control first. This new behavior differs quite a bit from previous .NET versions.

Until this bug has been fixed, the only workaround available is to use <AppContextSwitchOverrides value="Switch.UseLegacyToolTipDisplay=true"/> or AppContext.SetSwitch("Switch.UseLegacyToolTipDisplay", true);.

First opened here: https://github.com/microsoft/dotnet/issues/1065

**Comment by SamBent** [comment]
@rockerinthelocker - Please attach minimal repro(s) for your issues.  Your description contains at least one inaccuracy (there is no ShowTaskbar property), and several ambiguities ("closed programmatically", timing of minimize vs. tooltip display, etc.).  I can't repro either issue, even after guessing what you meant.

**Comment by rockerinthelocker** [comment]
@SamBent, You're right, it should have read `ShowInTaskbar` property.

However, since the issue affects advanced scenarios only (applications utilizing the [System.Windows.Forms] `NotifyIcon` class and hiding the window via `Window.Hide()` on minimize so the minimized window does not show up in the lower left corner of the desktop if `ShowInTaskbar` is set to `false`, utilizing the `DispatcherTimer` and `Tooltip` classes to control tooltip handling via keyboard and mouse programmatically, etc.), there might be no justification to fix this issue and developers simply have to opt-out via `AppContext.SetSwitch("Switch.UseLegacyToolTipDisplay", true);` for that type of applications.

If you feel the same, please close this issue. Please note that a repro would require hundreds of lines of code. Thank you!

**Comment by SamBent** [comment]
Closing per request.  There's not enough information here to be actionable.
